source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -195,6 +196,40 @@ + (NSNumber *)getTextStrokeWidthForWeight:(TextWeight)weight
             return @-2;
     }
 }
+// given string with short form of date (MM/dd/yyyy), convert it to
```

Comment:
>given string with short form of date (MM/dd/yyyy) [](start = 3, length = 49)

I don't think this comment correctly describes the format of the input string.

Reply:
that's correct, I need to update it. Thanks

---
In reply to: [171683925](https://github.com/Microsoft/AdaptiveCards/pull/1194#discussion_r171683925) [](ancestors = 171683925)